### PR TITLE
Enable MsSqlMigratorUtility Dockerfile to run with additional parameters

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Remodel Dockerfile in MsSqlMigratorUtility:
- Use `ENTRYPOINT` for the executable and main parameter
- Use `CDM` for additional parameters - to be able to exchange them when running in different mode via `docker run` 

## Code changes

* **util/MsSqlMigratorUtility/Dockerfile:** Description of what was changed and why

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 # DevOps for Actions and other workflow changes.
 .github/workflows @bitwarden/dept-devops
+util/MsSqlMigratorUtility/Dockerfile @bitwarden/dept-devops
```
Comment (withinfocus) [inline .github/CODEOWNERS:11]: 🎨 Check out https://github.com/bitwarden/mothership/blob/main/.github/CODEOWNERS#L13-L17 -- I think this is a stronger change.

**Review by withinfocus — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (michalchecinski) [reply .github/CODEOWNERS]: Thanks for the suggestion 😃

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -8,7 +8,12 @@
 
 # DevOps for Actions and other workflow changes.
 .github/workflows @bitwarden/dept-devops
-util/MsSqlMigratorUtility/Dockerfile @bitwarden/dept-devops
+
+# DevOps for Docker changes.
+**/Dockerfile @bitwarden/dept-devops
+**/*.Dockerfile @bitwarden/dept-devops
+**/.dockerignore @bitwarden/dept-devops
+**/prepare-env.sh @bitwarden/dept-devops
```
Comment (withinfocus) [inline .github/CODEOWNERS:16]: ❓ This file doesn't seem to exist anywhere -- should it be excluded?

Comment (michalchecinski) [reply .github/CODEOWNERS:16]: I thought adding it for the future. But removed it as you suggest.

**Review by withinfocus — APPROVED** [review]
Ownership change works for me. DevOps to provide final approval.

**Pull request merged**